tensorflow/python/eager/def_function.py

```diff
@@ -1281,6 +1281,10 @@ def function(func=None,
   `tf.function` constructs a `tf.types.experimental.GenericFunction` that
   executes a TensorFlow graph (`tf.Graph`) created by trace-compiling the
   TensorFlow operations in `func`.
+  
```

Comment:
nit: remove the w trailing whiltespaces?

Reply:
Done. I wish we had trailing space highlight on github. :)